### PR TITLE
Fix expo-av for usage with AndroidX

### LINKED ISSUE #5310: [expo-av] Android build fails with "cannot find symbol class Nullable"
### Environment

Bare React Native app with Expokit
react-native@0.60.4
react@16.8.6
expo-av@6.0.0
react-native-unimodules@0.5.4

### Steps to Reproduce

```
npm run android
```

### Expected Behavior

Should build project without issues.

### Actual Behavior

Build fails with the following error:

```
error: cannot find symbol class Nullable
```

I migrated my project to AndroidX and run jettifier on build. It doesn't matter if I build with Android Studio or via React Native CLI.

### Reproducible Demo

Sadly hard to share as it's directly tied to the build process.

### PR DESCRIPTION
**Pull request by lgraubner**
# Why

At the moment projects with expo-av do not compile for Android with AndroidX.

# How

The package is missing the "support-annotations" dependency in order to compile. After the change it runs fine using jettifier.

# Test Plan

Manually tested on Android Device and Simulator.

Fixes #5310, #5474  and unimodules/react-native-unimodules#61

### CODE REVIEW

File: packages/expo-av/android/build.gradle
Comment (sjchmiela) [inline packages/expo-av/android/build.gradle]:
```suggestion
  api "com.android.support:support-annotations:${safeExtGet("supportLibVersion", "28.0.0")}"
```

Let's use what we use everywhere else and conform to `supportLibVersion` setting

https://github.com/expo/expo/blob/eeecf97a3a36855daee8d6393508bedcd1c423cc/packages/expo-image-manipulator/android/build.gradle#L73

**Review by sjchmiela — APPROVED** [review]
LGTM! Thank you for taking care of this change. 💪  I'll wait for @mczernek's review before merging.

**Comment by Speedy1991** [comment]
Please be aware of my comment [here](https://github.com/expo/expo/issues/5310#issuecomment-529427984)

**Comment by lgraubner** [comment]
To make it clear: This is not a migration to AndroidX, but allows you to use jetifier to make it AndroidX compatible.

**Comment by superphil0** [comment]
This should be definitely mentioned here, took me 6 hours to find this pull request https://github.com/expo/expo/tree/master/packages/expo-av

**Pull request merged**
